doc/source/user_guide/io.rst

```diff
@@ -915,9 +910,7 @@ data columns:
 .. note::
 
    When passing a dict as the `parse_dates` argument, the order of
-   the columns prepended is not guaranteed, because `dict` objects do not impose
-   an ordering on their keys. On Python 2.7+ you may use `collections.OrderedDict`
-   instead of a regular `dict` if this matters to you. Because of this, when using a
+   the columns prepended is not guaranteed. Because of this, when using a
    dict for 'parse_dates' in conjunction with the `index_col` argument, it's best to
    specify `index_col` as a column label rather then as an index on the resulting frame.
```

Comment:
In Python 3.6 dicts should be ordered by insertion order, I'm not sure if this whole note should be removed or if the last sentence is still relevant.

Reply:
I *suppose* this is indeed no longer relevant, but would need to check (@gfyoung any idea?)

Reply:
OK, I removed it for now, can put it back if necessary. Thanks for the review, Joris!

Reply:
I think this removal is fine.